src/elements/file-selector/readme.md

```diff
@@ -84,6 +84,9 @@ It's suggested to have a different value for each variant, e.g.:
 ></sbb-file-selector>
 ```
 
+<!-- Override
```

Comment:
should we have this here although the logic is not in this PR?

Reply:
I would like to wait for the select to be ready anyway